site/gatsby-site/src/utils/cite.js

```diff
@@ -124,3 +124,71 @@ export const sortIncidentsByDatePublished = (incidentReports) => {
     }
   });
 };
+
+// Transforms the data from the graphql query into a History_incidentInsertInput format
+export const transformIncidentData = (incident, user) => {
+  const {
+    // eslint-disable-next-line no-unused-vars
+    __typename: incidentType,
+    AllegedDeployerOfAISystem,
+    AllegedDeveloperOfAISystem,
+    AllegedHarmedOrNearlyHarmedParties,
+    reports,
+    embedding,
+    nlp_similar_incidents,
+    tsne,
+    editors,
+    ...result
+  } = incident;
+
+  if (AllegedDeployerOfAISystem) {
+    result.AllegedDeployerOfAISystem = AllegedDeployerOfAISystem.link
+      ? AllegedDeployerOfAISystem.link
+      : AllegedDeployerOfAISystem.map((e) => e.entity_id);
+  }
+
+  if (AllegedDeveloperOfAISystem) {
+    result.AllegedDeveloperOfAISystem = AllegedDeveloperOfAISystem.link
+      ? AllegedDeveloperOfAISystem.link
+      : AllegedDeveloperOfAISystem.map((e) => e.entity_id);
+  }
+
+  if (AllegedHarmedOrNearlyHarmedParties) {
+    result.AllegedHarmedOrNearlyHarmedParties = AllegedHarmedOrNearlyHarmedParties.link
+      ? AllegedHarmedOrNearlyHarmedParties.link
+      : AllegedHarmedOrNearlyHarmedParties.map((e) => e.entity_id);
+  }
+
+  result.reports = reports ? reports.map((report) => report.report_number) : [];
+  result.nlp_similar_incidents = nlp_similar_incidents
+    ? nlp_similar_incidents.map((nlp) => {
+        // eslint-disable-next-line no-unused-vars
+        const { __typename: nlpType, ...currentNlp } = nlp;
+
+        return currentNlp;
+      })
+    : [];
+
+  if (embedding) {
+    // eslint-disable-next-line no-unused-vars
+    const { __typename: embeddingType, ...currentEmbedding } = embedding;
+
+    result.embedding = currentEmbedding;
+  }
+
+  if (tsne) {
+    // eslint-disable-next-line no-unused-vars
+    const { __typename: tsneType, ...currentTsne } = tsne;
```

Comment:
I think if you set it to `undefined` you don't need to disable the linter

Reply:
Fixed